rmw_opendds_cpp/include/rmw_opendds_cpp/opendds_static_publisher_info.hpp

```diff
@@ -50,6 +50,36 @@ class OpenDDSPublisherListener : public DDS::PublisherListener
     return current_count_;
   }
 
+  void on_offered_deadline_missed(
+    ::DDS::DataWriter_ptr writer,
```

Comment:
Do we need to start from the global namespace? Can't it just be `DDS::DataWriter_ptr writer`

Reply:
Fixed